src/app.rs

```diff
@@ -691,17 +684,23 @@ fn save_using_target(cx: &mut Context, desc: TableDescription) -> Result<(), Dyn
 
     // write to config file
     let config_yaml_string = serde_yaml::to_string(config)?;
-    fs::write(
-        retrieve_dynein_file_path(DyneinFileType::ConfigFile)?,
-        config_yaml_string,
-    )?;
+    write_dynein_file(DyneinFileType::ConfigFile, config_yaml_string)?;
 
     // save target table info into cache.
     insert_to_table_cache(cx, desc)?;
 
     Ok(())
 }
 
+fn write_dynein_file(dft: DyneinFileType, content: String) -> Result<(), DyneinConfigError> {
```

Comment:
I could not infer the actual meaning from `dft` without checking the type. How about `file_type` simply instead of `dft`.